spec-0009/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "SPEC 9 — Governance"
+number: 9
+date: 2024-06-04
+author:
+  - "Sanket Verma <svsanketverma5@gmail.com>"
+discussion: https://discuss.scientific-python.org/t/spec-9-governance/1229
+endorsed-by:
+---
+
+## Description
+
+Open-source project communities are amalgamations of software and the humans who
+use and maintain it. We have various systems, like governance models, which
+define certain roles for participating members in the project, and a code of
+conduct, which protects and fosters a safe and collaborative environment for
+the community.
+
+Having a robust system in place ensures the long-term sustainability of the
+project.
+
+The OS communities interact and contribute through various mediums, such as
+GitHub, a chat platform, social media, blog posts, etc., and having guidelines
```

Comment:
Thanks for the suggestion here, @mdhaber.

I'd like to keep 'etc.' here as I've left out some of the mediums through which the community interacts with the project. For example, forums like Discourse, mailing lists, and Zoom meetings (community calls, office hours, etc.).

What do you think about this:

The OS communities interact and contribute through various mediums, _like_ distributed version control systems, chat platforms, social media, blog posts, etc. Having clear guidelines for participation helps maintain a continuous stream of contributions.

Reply:
See, for example:
https://www.grammarbook.com/blog/definitions/all-about-etc/

> And never use etc. at the end of a series that begins with for example, e.g., including, such as, and the like, because these terms make etc. redundant: they already imply that the writer could offer other examples.

This is the point I was making. I think "like" also indicates that the list is incomplete, so "etc." is redundant.

Reply:
TIL. Thanks!
Done. Check here: https://github.com/scientific-python/specs/pull/323/commits/5b5def515c2d31e15b6432a21d7a3555975ba839